lib/main.dart

```diff
@@ -3,14 +3,22 @@ import 'dart:io';
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 import 'package:image_picker/image_picker.dart';
-import 'package:photo_hooks/database/dbhelper.dart';
-import 'package:photo_hooks/model/encoding_and_decoding_for_picture.dart';
-import 'package:photo_hooks/screens/albums_screen.dart';
-import 'package:photo_hooks/screens/galery_screen.dart';
-import 'package:photo_hooks/model/picture.dart';
-import 'package:photo_hooks/screens/photo_view_screen.dart';
-
-void main() {
+
+import './presentation/screens/albums_screen.dart';
```

Comment:
change paths to package like - 'package:photo_hooks/...' etc, as you had it previously